src/models/Organization.ts

```diff
@@ -84,6 +87,12 @@ const organizationSchema = new Schema(
       ref: "User",
       required: true,
     },
+    actionCategories: [
```

Comment:
get rid of this, relationship to an organization is already present on `Category` using `organizationId` field